contributing.md

```diff
@@ -0,0 +1,54 @@
+## Becoming a Maintainer
+
+The documentation for becoming a maintainer has been taken from [Foreman](https://theforeman.org/handbook.html#Becomingamaintainer) and adapted for the PatternFly project.
+
+As a maintainer, it is your responsibility to help manage and maintain the health of the PatternFly project.  A PatternFly maintainer has commit permissions to one or more of the repositories under the [PatternFly organization](https://github.com/patternfly). To see a list of maintainers to the PatternFly project, view the [PatternFly GitHub Teams](https://github.com/orgs/patternfly/teams) page.
+
+Maintainers are members of the PatternFly community who exhibit most of the following behaviors:
+
+- Review and merge code and documentation.
+- Help triaging bugs and testing pull requests.
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others.
+- Encourage and ensure design remains an integral part of the review process and pull in designers for review as needed (you can leverage @patternfly/angular-patternfly-ux if there is no known associated designer).
+- Maintain sustained activity versus sporadic.
+- Support users and other developers on [PatternFly Slack](https://patternfly.slack.com/) (there is a channel dedicated to Angular Patternfly) and the [mailing list](mailto:patternfly@redhat.com).
+
```

Comment:
mailto:angular-patternfly@redhat.com

Reply:
Thanks @dtaylor113 will make that change, @LHinson are there other mailing lists for the other repos that I need to be aware of? I didn't update this for the other ones.

Reply:
We can add the angular-pf ML. The angular ML is pretty much inactive. I'd also include the PF ML since it is more active. None of the other repos have MLs as we didn't want to continue to divide the community as it didn't seem like a successful approach.

Reply:
How should this be reworded to include both?

(there is a channel dedicated to Angular Patternfly) and both the [angular patternfly mailing list](mailto:patternfly@redhat.com) and the [patternfly mailing list](mailto:patternfly@redhat.com).

Reply:
- Support users and other developers on [PatternFly Slack](https://patternfly.slack.com/) as well as our mailing lists, Angular Patternfly (
mailto:angular-patternfly@redhat.com) and PatternFly (mailto:patternfly@redhat.com).